File: 2390 removing-stars-from-a-string/solution.cs
public class Solution {
    public string RemoveStars(string s) {
        char[] st = new char[s.Length];
        int top = -1;
        foreach(char c in s) {
            if(top!=-1 && c=='*') top--;
            else if(c!='*') st[++top] = c;
        }
        return new String(st,0,top+1);
    }
}

File: 2390 removing-stars-from-a-string/solution.go
func removeStars(s string) string {
        st := make([]rune, len(s))
        top := -1
        for _, c := range s {
            if top!=-1 && c=='*' {
                top--
            } else if c!='*' {
                top++
                st[top] = c
            } 
        }
        return string(st[:top+1])
}
